index.bs

```diff
@@ -1900,7 +1859,10 @@ engine.
     :: The TPMT_PUBLIC structure (see [[TPMv2-Part2]] section 12.2.4) used by the TPM to represent the credential public key.
 
 : Signing procedure
-:: Concatenate the given <a>authenticatorData</a> and <a>clientDataHash</a> to form |attToBeSigned|.
+:: Let |authenticatorData| denote the [=authenticator data for the attestation=], and let |clientDataHash| denote the
```

Comment:
This makes it look like the authenticator needs to compute the authenticator data and the hash of the serialized client data, when actually they're just arguments to this function. I think it'd be clearer to write the signing procedure as a function with parameters.